cmake/zjs_linux.cmake

```diff
@@ -140,9 +140,13 @@ if(NOT APPLE)
 
   if("${DEBUGGER}" STREQUAL "on")
     list(APPEND APP_SRC
-      ${JERRY_BASE}/jerry-port/default/default-debugger-transport.c
+      ${JERRY_BASE}/jerry-core/debugger/debugger-sha1.c
+      )
+    list(APPEND APP_INCLUDES
+      ${JERRY_BASE}/jerry-core/ecma/base
+      ${JERRY_BASE}/jerry-core/jmem
+      ${JERRY_BASE}/jerry-core/lit
```

Comment:
I still build linux debugging mode failed with this patch, it report that `src/zjs_console.c:16:22: fatal error: debugger.h: No such file or directory`,  `${JERRY_BASE}/jerry-core/debugger` is needed here. I just test it work, you could have a double confirm.